packages/tinacms/src/components/media/media-manager.tsx

```diff
@@ -237,6 +237,24 @@ export function MediaPicker({
     }
   }
 
+  type CustomAction = {
+    name: string,
+    onClick: (props: any) => void
```

Comment:
How about `CustomActionProps` for types instead of any?

Reply:
Good idea